infrastructure/services.tf

```diff
@@ -0,0 +1,32 @@
+# common services resources
+
+resource "azurerm_resource_group" "services_rg" {
+  name     = var.services_rg_name
+  location = var.location
+}
+
+resource "azurerm_dns_zone" "public_dns_zone" {
+  name                = var.dns_domain
+  resource_group_name = azurerm_resource_group.services_rg.name
+}
+
+resource "azurerm_container_registry" "acr" {
+  name                    = var.acr_name
+  resource_group_name     = azurerm_resource_group.services_rg.name
+  location                = azurerm_resource_group.services_rg.location
+  sku                     = "Standard"
+  admin_enabled           = false
+  anonymous_pull_enabled  = false
+  data_endpoint_enabled   = false
+  zone_redundancy_enabled = false
+}
+
+resource "azurerm_key_vault" "akv" {
```

Comment:
Does the application insights instrumentation key get stored as a Key Vault secret?